coredb-operator/yaml/sample-coredb.yaml

```diff
@@ -5,7 +5,6 @@ metadata:
 spec:
   replicas: 1
   extensions:
-    - name: "pg_stat_statements"
-      enabled: true
-      version: "1.9"
-      schema: "postgres"
+    - name: pg_stat_statements
+      locations: 
+        - enabled: true
```

Comment:
Nitpick - I like including the extra spec values such as `database`, `schema`, `version` in the sample. It could help a newcomer better understand what the `locations` value is for.

Reply:
I just took these out per @sjmiller609 request

Reply:
brought them back in!